tests/integration/targets/kubevirt_vm/verify.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Wait for ssh
+  connection: local
+  gather_facts: false
+  hosts: localhost
+  tasks:
+    - name: Connectivity
+      block:
+        - name: Wait 600 seconds for port 22 to become open
+          ansible.builtin.wait_for:
+            port: 22
+            host: "{{ hostvars['default-testvm3'].ansible_host }}"
+            delay: 30
+            timeout: 600
+      rescue:
+        - name: Port 22 not open
+          ansible.builtin.debug:
+            msg: "Port 22 still not open"
+        - name: Wait 300 seconds for port 22 to become open
+          ansible.builtin.wait_for:
+            port: 22
+            host: "{{ hostvars['default-testvm3'].ansible_host }}"
+            delay: 30
+            timeout: 300
+
+- name: Connect to VM
+  gather_facts: true
+  hosts: default-testvm3
+  remote_user: fedora
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Print out virtual machine facts
+      ansible.builtin.debug:
+        var: ansible_facts
+
+- name: Verify creation with existing VM
+  connection: local
```

Comment:
Is this line needed?

Reply:
it is not necessarily needed, but in a playbook file with mixed plays that both use the inventory or just localhost it is a good practice to be explicit